### PR TITLE
V pedro create weapons

### PR DESCRIPTION
**Pull request by PedroBarge**

### CODE REVIEW

File: src/Weapons/AddWeapons.java
Comment (rodgreg02) [inline src/Weapons/AddWeapons.java:1]: change the class name you imbecile

File: src/Main.java [hunk 1]
```diff
@@ -1,5 +1,12 @@
+import Weapons.AddWeapons;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //Story.startStory();
+        //Story.goodEnding();
+        //Story.baddEnding();
```
Comment (rodgreg02) [inline src/Main.java:7]: badd???? are you dumb bruv

**Review by rodgreg02 — CHANGES_REQUESTED** [review]
change names, that it lol

File: src/Main.java (hunk 1 above)
Comment (tataket) [reply src/Main.java:7]: badd????wtf brooooo

**Review by rodgreg02 — APPROVED** [review]
very good

**Pull request merged**